server/src/services/conversion_service.ts

```diff
@@ -52,8 +60,10 @@ export class ConversionService {
 
     const spotifyTrack = await this.spotifyService.getTrackInfo(id)
 
+    const artistsString = spotifyTrack.artists.join(', ')
```

Comment:
```suggestion
    const artistsString = spotifyTrack.artists.join(' ')
```
empty space instead of a comma?